src/treq/client.py

```diff
@@ -431,6 +448,29 @@ def _guess_content_type(filename):
     return guessed or 'application/octet-stream'
 
 
+def _get_content_type_from_header(headers, default=None):
+    """
+    Retrieve Content-Type header from twisted.web.http_headers.Headers object
+    or return supplied default
+
+    :param headers:
+        A twisted.web.http_headers.Headers object
+
+    :param default:
+        The default Content-Type to return, encoded as a byte-string
+
+    :returns:
+        byte-string containing a Content-Type value
+    :rtype:
+        bytes
+    """
+    if not headers:
```

Comment:
At this point *headers* is always a `Headers` object:

https://github.com/twisted/treq/blob/d45cad202a5b4650213c170725b2a0b0577f94ee/src/treq/client.py#L174-L188

This clause is dead code and should be removed (at which point this function doesn't do much...)